### DISCUSSION TITLE
Remotewrite data from mimir ingester to another system

### DISCUSSION THREAD

**Discussion by RaviHari** [grafana/mimir#8051] [Help and support]
I want to be able to write data from mimir ingester to another system via remotewrite protocol.
Does mimir support it?

**Comment by charleskorn** [comment]
This is not supported. Could you explain a little more about why you want to do this? There might be another way to achieve what you want.

**Reply by RaviHari** [reply]
I would like to send metrics from Mimir to an event based system.
I am not using Prometheus.. so for derived metrics I am looking for a solution with Mimir ecosystem.. if I write data from OTel to Mimir.. how can we send Metrics with a regex pattern, derived metrics emitted from Mimir ruler to another system.

**Reply by charleskorn** [reply]
I don't think Mimir is the right tool for what you're trying to do unfortunately.
